src/main/resources/db/migration/V8__sharing_options_add.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE calendar_id
+    ADD sharing_title BOOL,
+    ADD sharing_location BOOL,
+    ADD sharing_description BOOl,
+    ADD sharing_organizer BOOL,
+    ADD sharing_attendants BOOL;
```

Comment:
V8 has been created during refactor so this should be V9 (and calendar_id -> google_calendar)